json-schemas/iva/defects/get/index.json

```diff
@@ -1,12 +1,11 @@
 {
-	"title": "Defect GET IVA",
+	"title": "Section IVA",

```

Comment:
As per your comments, i'm happy with this name for now. 

I know there will be further tickets to refactor everything done so it is more in line with the correct term of 'Required Standard(s)'.